package.json

```diff
@@ -25,12 +25,12 @@
   },
   "devDependencies": {
     "@types/jest": "^24.0.11",
-    "@typescript-eslint/eslint-plugin": "^1.5.0",
+    "@typescript-eslint/eslint-plugin": "^4.1.1",
     "@typescript-eslint/parser": "^4.1.1",
-    "eslint": "^5.15.3",
-    "eslint-config-prettier": "^4.1.0",
-    "eslint-plugin-import": "^2.16.0",
-    "eslint-plugin-prettier": "^3.0.1",
+    "eslint": "^7.9.0",
```

Comment:
With this bump you can remove the `--ext` flag to `yarn lint` script. Since we have an override for `**/*.ts`, all `.ts` files will automatically get linted (this was one of the nicer changes in 7.0.0 IMHO).

Reply:
Oh, good to know. Removed that!

Reminds me of my early TS endeavors, being glad that linting immediately passes without any errors, just to realize later that it was not even running on .ts files! 😝